module/accelerometer/include/accelerometer.hpp

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) 2023 Robarm
+// All rights reserved
+#ifndef MODULE_ACCELEROMETER_INCLUDE_ACCELEROMETER_HPP_
+#define MODULE_ACCELEROMETER_INCLUDE_ACCELEROMETER_HPP_
+
+#include "hal/i2c/include/i2c_component.hpp"
+
+struct AccelerationRAW_t {
+    int16_t x;
+    int16_t y;
+    int16_t z;
+};
+
+struct Acceleration_t {
+    float x;
+    float y;
+    float z;
+};
+
+namespace robarm {
+namespace module {
+namespace accelerometer {
+class Accelerometer {
```

Comment:
Separate namespaces from classes in header file.

```suggestion
namespace accelerometer {

class Accelerometer {
```